dev_tools/requirements/deps/ipython.txt

```diff
@@ -1,3 +1 @@
-# https://github.com/ipython/ipython/issues/12941
-ipython==7.31.1; python_version >= '3.7'
-ipython; python_version < '3.7'
+ipython
```

Comment:
Can we just pin the major version?  That would exclude IPython 7.32 that was initially excluded.

```suggestion
ipython~=8.0.0
```

Reply:
Good idea. Done.